tests/Pizza.go

```diff
@@ -0,0 +1,78 @@
+// Code generated by the FlatBuffers compiler. DO NOT EDIT.
+
+package Pizza
```

Comment:
Package names should be all lowercase.

See, https://go.dev/blog/package-names

Reply:
@enum-class can you fix

Reply:
Thank you @le-michael.
 @dbaileychess I am on it

Reply:
Hi @le-michael, What should generator do in scenarios that namespace in fbs file is not "Good package names" ?
should we expect that generator change it to something Good ? or do not care about "Good package names" rules and respect the fbs file ?
cc: @dbaileychess

Reply:
@le-michael , In master branch if no namespace defined, the generator put empty string (") as package name which is not correct. In this p.r. I have been using filename as package name. Do you think is it good idea at all to use filename as package name ?

Reply:
Looks like the existing implementation doesn't care about the package name format either so this might not be a problem.

For example,
https://github.com/google/flatbuffers/blob/master/tests/MyGame/Example/Monster.go#L3

> What should generator do in scenarios that namespace in fbs file is not "Good package names" ?
should we expect that generator change it to something Good ? or do not care about "Good package names" rules and respect the fbs file ?

Ideally, we should update the namer config so that the generated code matches the [style guide](https://go.dev/doc/effective_go#names), but this might break client projects when they update their flatbuffer version.

I'll defer to @dbaileychess on what we should do here.

> In master branch if no namespace defined, the generator put empty string (") as package name which is not correct. In this p.r. I have been using filename as package name. Do you think is it good idea at all to use filename as package name ?

Filename as package name makes sense to me. In the case that we have multiple tables in the same file with no namespace having them in the same go package lets them reference each other without imports.

Reply:
BTW, by file name I am referring to the `*.fbs` name not the generated Go file. 

For example,

```
// example.fbs

table Foo {
  bar: Bar;
}

table Bar {
  id:uint32;
}
```

should be generated under the same go package.

```
// Foo.go
package example

type Foo struct {
...
}
...
```

```
// Bar.go
package example

type Bar struct {
...
}
...
```

Reply:
There is a legacy naming function in the namer where we can localize these incompatibilities. I would just file a bug and introduce this name format as a legacy naming function and we can address later.